bin/enumerate_team_members.dart

```diff
@@ -124,13 +124,17 @@ void main(List<String> args) async {
     dynamic earliestAuthored;
     try {
       earliestAuthored = await github.searchIssuePRs(earliestQueryAuthor).first;
-    } catch (e) {}
+    } catch (e) {
```

Comment:
Since `e` is ignored, consider renaming it `_`.

Reply:
Thanks! Updated.